js/base/Exchange.js

```diff
@@ -1523,7 +1523,7 @@ module.exports = class Exchange {
         for (let i = 0; i < symbols.length; i++) {
             result.push (this.symbol (symbols[i]));
         }
-        return result;
+        return (result.length === 0) ? undefined : result;
```

Comment:
@pcriadoperez we can't do this, array's length has to be in a separated line otherwise will transpile to string length.

```Javascript
const resultLength = result.length;
(...)
```